### PR TITLE
#998: fixed release-info messed with installation output

### PR DESCRIPTION
**Pull request by Amueller36**
Added mvn setup call at the beginning of the command as stated by @hohwille. I couldn't reproduce the error so I assume that might be it.

### CODE REVIEW

**Comment by Amueller36** [comment]
#998 related issue

File: .mvn/maven.config [hunk 1]
```diff
@@ -1 +1 @@
--Drevision=2023.01.001-SNAPSHOT
+-Drevision=2023.01.001
```
Comment (hohwille) [inline .mvn/maven.config:1]: you should not commit changes to this file.

**Review by hohwille — CHANGES_REQUESTED** [review]
@Amueller36 Thanks for this PR. Your fix is fine👍 
However, can you revert the change to `maven.config` please?

**Review by hohwille — APPROVED** [review]
@Amueller36 thanks for your PR. Workaround should fix the problem though it will install maven also if the release build will not use maven...

**Pull request merged**